code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1048,6 +1048,9 @@
 		else if(istype(W, /obj/item/shard/shrapnel))
 			var/obj/item/shard/shrapnel/embedded = W
 			embedded.on_embedded_movement(src)
+		else if(istype(W, /obj/item/sharp))
+			var/obj/item/sharp/embedded = W
+			embedded.on_embedded_movement(src)
```

Comment:
all this istyping is mid and these 3 objs should use proper inheritance so this single proc can be called on all of them :3

Reply:
But erm largesharp arent shards and nor is this 💀